### DISCUSSION TITLE
Support Binance Smart Chain Accounts

### DISCUSSION THREAD

**Discussion by cryptopolisrex** [LedgerHQ/ledger-live-desktop#4012] [Coin/Token request]
- [X] I have checked this feature was not yet requested.

Currently there is no way ( I know of) to check the status / balance of accounts on the Binance Smart Chain in ledger live. Would be nice to add. 

#### Part of the application

Accounts

**Comment by akanass** [comment]
Totally agree with this one and Binance Chain account as well like this we will able to see balance for BEP2 and BEP20 tokens

**Comment by leeweezee777** [comment]
Yes please implement this.  This filthy girl has jumped onto Binance Smart Chain.

**Comment by franco999** [comment]
Any news about this? I would really like to see the equivalence of the balance in local currency and currently this is not happening.

**Comment by ceskmcfran** [comment]
Would be a really good feature if Ledger can give you any BEP2 address to store your BNB (BEP2 network) tokens. There's a lot of projects where you need to pay fees with bep2 and I want to use them with the security of Ledger.

**Comment by dalechyn** [comment]
Absolute bump to this one
